easybuild/easyconfigs/j/jq/jq-1.5-foss-2016a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'jq'
+version = '1.5'
+homepage = 'https://stedolan.github.io/jq/'
+description = """jq is a lightweight and flexible command-line JSON processor."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/stedolan/jq/archive/']
+sources = ['%(name)s-%(version)s.tar.gz']
+
+dependencies = [('Bison', '3.0.4')]
+
+sanity_check_paths = {
+    'files': ["lib/libjq.so.1.0.4", 
+              "lib/libjq.a", 
+              "lib/libjq.la",
+              "bin/jq"],
```

Comment:
no need to spread these across different lines imho; also, prefer only checking the `.so`, albeit via `SHLIB_EXT`:

```python
'files': ['bin/jq', 'lib/libjq.a', 'lib/libjq.%s' % SHLIB_EXT],